server/td-agent.conf

```diff
@@ -0,0 +1,48 @@
+<system>
+  log_level debug
+</system>
+
+## match tag=debug.** and dump to console
+<match debug.**>
+  @type stdout
+  @id output_stdout
+</match>
+
+####
+## Source descriptions:
+##
+
+## built-in TCP input
+## @see http://docs.fluentd.org/articles/in_forward
+<source>
+  @type forward
+  <transport tls>
+    cert_path /etc/td-agent/certs/fluentd.crt
+    private_key_path /etc/td-agent/certs/fluentd.key
+    private_key_passphrase PASSPHRASE_HERE
+  </transport>
+  <security>
+    self_hostname logz-dev
+    shared_key SHARED_KEY_HERE
```

Comment:
Same question here... can we use an env var?

Reply:
Can?  Yes.  Should?  Questionable -- see above.